lib/config/presets/internal/replacements.ts

```diff
@@ -91,6 +93,32 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  'react-query-devtools-to-scoped': {
+    description:
+      '`react-query/devtools` became scoped under the `tanstack` organization.',
+    packageRules: [
+      {
+        matchCurrentVersion: '>=2.6.3 <4.0.0',
+        matchDatasources: ['npm'],
+        matchPackageNames: ['react-query/devtools'],
+        replacementName: '@tanstack/react-query-devtools',
+        replacementVersion: '4.0.0',
+      },
+    ],
+  },
+  'react-query-to-scoped': {
+    description:
+      '`react-query` became scoped under the `tanstack` organization.',
+    packageRules: [
+      {
+        matchCurrentVersion: '>=3.39.2 <4.0.0',
```

Comment:
Same question here, do we want people to be on the latest `non-scoped` version before we offer the replacement PR. Or do we just want them to be on any `3.x.x` version?

Reply:
```suggestion
        matchCurrentVersion: '>=3.0.0 <4.0.0',
```

Sounds like we want this then?